test/appium/tests/api_requests.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import pytest
+import requests
+import time
+
+
+def get_transactions(address: str) -> dict:
+    url = 'http://ropsten.etherscan.io/api?module=account&action=txlist&address=0x%s&sort=desc' % address
+    return requests.request('GET', url=url).json()['result']
+
+
+def is_transaction_successful(transaction_hash: str) -> int:
+    url = "https://ropsten.etherscan.io/api?module=transaction&action=getstatus&txhash=%s" % transaction_hash
+    return not int(requests.request('GET', url=url).json()['result']['isError'])
+
+
+def get_balance(address):
+    url = 'http://ropsten.etherscan.io/api?module=account&action=balance&address=0x%s&tag=latest' % address
+    for i in range(5):
+        try:
+            return int(requests.request('GET', url).json()["result"])
+        except ValueError:
+            pass
+
+
+def find_transaction_on_ropsten(address: str, transaction_hash: str):
+    transactions = get_transactions(address=address)
+    for transaction in transactions:
+        if transaction['hash'] == transaction_hash:
+            logging.info('Transaction is found in Ropsten network')
+            return
+    pytest.fail('Transaction is not found in Ropsten network')
+
+
+def verify_balance_is_updated(initial_balance, recipient_address, wait_time=240):
+    counter = 0
+    while True:
+        if counter == wait_time:
```

Comment:
I would go with >= in this conditional statement to eliminate possibility of endless loop in "while"